src/guides/textSplitting.md

```diff
@@ -1,86 +1,165 @@
 # Text Splitting
 Text splitting functions are useful for advanced codes that deal with multiple user arguments, or even adjusting function outputs *(for advanced users)*. This wiki includes information on how to use these functions.
 
-## Functions
-### $textSplit[]
+# Functions
+There are seven functions in Text Splitting. One of them - $textSplit - is the main function that you start to work with Text Splitting.
```

Comment:
This sentence doesn't seem to be grammatical

Reply:
```suggestion
There are 7 functions in Text Splitting. One of them is `$textSplit`, which starts all the use of Text Splitting, as well as its study.
```

Reply:
I would go with something like this instead:
```suggestion
There are seven functions in this guide. One of them, `$textSplit`, is the main function that you start off 
```

Reply:
```suggestion
There are 7 functions in this guide. One of them, `$textSplit`, is the main function that you start off.
```